### ISSUE TITLE
Docker test build workflow is broken

### ISSUE THREAD

**Issue by zeldin** [ps3dev/ps3toolchain#96]
The Docker does not install libgmp-dev, so the build always bails out immediately...

**Comment by miigotu** [comment]
Yep I know, took a break from it because it was irritated with it lol

**Comment by miigotu** [comment]
It actually does install libgmp-dev (and also trying with libgmp3-dev meta package) and it still fails and i cant figure out why.

**Comment by miigotu** [comment]
I got it building just having an issue with pushing it now. Not sure if pkg.github.com likes multi-arch manifests, and I dont have rights to add a secret key for hib.docker.com login here on ps3dev.

Ill make some more attempts.

**Comment by miigotu** [comment]
Wow I totally forgot it wasn't working. check-gmp just isn't finding the headers on xenial (libgmp-dev is installed)
Working on it now.

Do you think I should change the base image from xenial to focal and get it building there? I think current LTS should always be our target if using ubuntu.

**Comment by zeldin** [comment]
Well, https://github.com/ps3dev/ps3dev-docker is using bionic (18.04 LTS) as base image, and that seems to work well.  I've also succefully used debian:10-slim.

**Comment by miigotu** [comment]
I think debian 10 slim is a great option actually. I'll see if I can get that building. It's currently having a protocol error with the manifest 🤦‍♂️🙄

**Comment by zeldin** [comment]
I just uploaded fresh builds of ps3dev-docker to [docker hub](https://hub.docker.com/repository/docker/zeldin/ps3dev-docker) based on 10-slim (the ppc64 build is based on sid-slim instead because ppc64 is currently an "unofficial" Debian port and does not get stable builds).  If you agree that 10-slim is good I guess I'll push the change of base image to ps3dev-docker.

**Comment by humbertodias** [comment]
Try out https://github.com/ps3dev/ps3dev-docker/pull/6
There the flow of deploying to [docker hub](https://hub.docker.com/r/hldtux/ps3dev) was fixed

**Comment by BriellaBugs** [comment]
http://xmlsoft.org/download/libxml2-2.7.8.tar.gz 404

**Comment by BriellaBugs** [comment]
https://xmlsoft.org/ftp/libxml2/libxml2-2.7.8.tar.gz seems to be the current alternative
